showroom/admin.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib import admin
+from .models import *
```

Comment:
Try not to use relative imports, instead use absolute paths
Separate models by apps and import them by that example: `from showroom.models import Showroom`